db-engine/src/main/java/com/splicemachine/db/impl/sql/conn/SQLSessionContextImpl.java

```diff
@@ -106,4 +106,8 @@ public SchemaDescriptor getDefaultSchema() {
     }
 
     public List<String> getCurrentGroupUser() { return groupuserlist; }
+
+    public void setCurrentGroupUser(List<String> groupUsers) {
+        groupuserlist = groupUsers;
```

Comment:
It's getting worse every change - the name of the setter is in singular while the input argument is in plural and then the internal variable is in singular with the type in its name :(